packages/mongo-menu/src/MongoMenu.spec.tsx

```diff
@@ -18,9 +18,17 @@ describe('packages/MongoMenu', () => {
     />,
   );
 
+  const { getByText } = renderedComponent;
   const trigger = getByText('Leafy');
   fireEvent.click(trigger);
 
+  test('Account button links to the default account app', () => {
+    const accountItem = getByText('MongoDB Account').parentElement;
+    expect((accountItem as HTMLAnchorElement).href).toBe(
```

Comment:
I might be missing something, but is there a reason we're explicitly typing accountItem as `HTMLAnchorElement`? If so, might it be better to do it on the line above like so?

```ts
const accountItem: HTMLAnchorElement = getByText('MongoDB Account').parentElement;
```

Reply:
`getByText('MongoDB Account')` will always return a value (or throw an error), but the call to `parentElement` can be null, and otherwise returns an `HTMLElement`

You cannot set `accountItem` to `HTMLAnchorElement` for two reasons:
(a) it could be null, which is fine, we can get around that by just adding `| null` or something similar
(b) HTMLElement is missing all of the unique properties that are expected on `HTMLAnchorElement`, which we cannot take care of easily.

As a result, we have to explicitly cast. Presumably I can cast earlier though:
```
const accountItem = getByText('this test').parentElement as HTMLAnchorElement | null;
expect(accountItem.href).toBe('wow this is not mocha/chai');
```
However to be honest, I did this to keep with the pattern already established in this test.